ghtt/search.py

```diff
@@ -84,7 +84,7 @@ def search(ctx, query, mg_api_key, mg_domain, to):
         click.secho(g_repo.html_url, fg="red")
 
         click.secho("Metadata of last commit:")
-        g_commit = g_repo.get_branch("master").commit
+        g_commit = g_repo.get_branch(g_repo.master_branch or "master").commit
```

Comment:
```suggestion
        g_commit = g_repo.get_branch(g_repo.default_branch).commit
```

Reply:
Ok, didn't see that `g_repo.default_branch` exists. I only saw `g_repo.master_branch` which doesn't exisit in the GitHub API apparently :man_shrugging: 
Anyway, `g_repo.default_branch` is better indeed!